README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_growth_table_DOM/)
+    - [DEMO LINK](https://SerhiiSemennikov.github.io/js_growth_table_DOM/)
```

Comment:
Add demo link to the PR description 
<img width="945" alt="image" src="https://github.com/user-attachments/assets/26773949-3b4f-4652-a202-900ffb173dbb">